src/lib/internal-icons.js

```diff
@@ -488,6 +488,13 @@ const icons = {
       </g>
     </g>`,
   },
+  help: {
```

Comment:
Did you minfy the `svg` ? If not, whenever you add svgs, minify first, [a great site that I use](https://jakearchibald.github.io/svgomg/).

Reply:
Yeah, and, if its possible, in this case, you just need the `path` tag inside the `g` tag